tests/sentry/integrations/discord/test_utils.py

```diff
@@ -106,3 +107,41 @@ def test_not_supported_type(self, mock_get_channel):
             validate_channel_id(
                 self.channel_id, self.guild_id, self.integration_id, self.guild_name
             )
+
+
+class GetChannelIdFromUrl(TestCase):
+    def test_happy_path(self):
+        channel_id = get_channel_id_from_url("https://discord.com/channels/guild-id/channel-id")
```

Comment:
should these be integers instead of strings to match reality?

Reply:
They are strings in reality since it is user input